docs/basics/basic_scenario.md

```diff
@@ -0,0 +1,56 @@
+# Scenario
+
+Here is the basic scenario structure.  There is only one Scenario for one phase.  Each scenario consists of sequences.  When many sequences are specified in below synstax, they are executed sequentially.  If parallel execution are required, please see [initialSequences and sequences]({{ "/userguide/benchmark/scenario.html" | absolute_url }}).  For more detail, please see [scenario]({{ "/docs/concepts.html#scenario" | absolute_url }})
+
+Note: httpRequest is a **step**, it will be the same level as all the other STEPS.
+
+```yaml
+scenario:
+- SEQUENCE_NAME:
+  (Optional - STEPS)
+  - httpRequest: ...
+```
+
+Example: 
+
+```yaml
+scenario:
+- queryFlight:
+  - thinkTime:
```

Comment:
I know this is only for demonstration purposes, but the `thinkTime` here does not make much sense. What about firing two `httpRequest`s (to different paths, or a GET and POST...) and have the think time in between?

Reply:
I have never thought of having 2 httpRequest here..  quickstart has more complex benchmarks & I would like to have as simple as possible here. let me change it to something else than thinkTime.

Reply:
In the quickstart it gets a bit more complicated as the requests are async. But sure, feel free to pick a different example, I'll just voice my concerns.